lm-logs/values.yaml

```diff
@@ -24,6 +24,9 @@ fluent:
       chunk_limit_size: 8m
       flush_thread_count: 8
 
+kubernetes:
+  multiline_start_regexp: /^\[(\d{4}-)?\d{2}-\d{2} \d{2}:\d{2}:\d{2}.\d{3}.*\]/
```

Comment:
We should not keep a default, It is upon customer to customer what they want to add it as.